frame/system/src/lib.rs

```diff
@@ -798,6 +798,14 @@ pub struct EventRecord<E: Parameter + Member, T> {
 pub enum RawOrigin<AccountId> {
 	/// The system itself ordained this dispatch to happen: this is the highest privilege level.
 	Root,
+	/// Sudo Governance Token layer.
+	Gover,
+	/// Sudo Update layer.
+	Updater,
+	/// Sudo Calls layer.
+	Sudoer,
+	/// Sudo Pause Transactions layer.
+	Pauser,
```

Comment:
Remove these because:
* `Sudoer` - already exists via `Root`
* `Pauser` - not possible

Reply:
> Remove these because:
> 
> * `Sudoer` - already exists via `Root`
> * `Pauser` - not possible

you are right! pushing now